io_scene_dae/export_dae.py

```diff
@@ -1503,32 +1572,32 @@ def export_scene(self):
             if (obj in self.valid_nodes and obj.parent is None):
                 self.export_node(obj, 2)
 
-        self.writel(S_NODES, 1, '</visual_scene>')
-        self.writel(S_NODES, 0, '</library_visual_scenes>')
+        self.writel(S_NODES, 1, "</visual_scene>")
+        self.writel(S_NODES, 0, "</library_visual_scenes>")
 
     def export_asset(self):
-        self.writel(S_ASSET, 0, '<asset>')
+        self.writel(S_ASSET, 0, "<asset>")
         # Why is this time stuff mandatory?, no one could care less...
-        self.writel(S_ASSET, 1, '<contributor>')
+        self.writel(S_ASSET, 1, "<contributor>")
         # Who made Collada, the FBI ?
-        self.writel(S_ASSET, 2, '<author> Anonymous </author>')
+        self.writel(S_ASSET, 2, "<author> Anonymous </author>")
         # Who made Collada, the FBI ?
         self.writel(
-            S_ASSET, 2, '<authoring_tool> Collada Exporter for Blender 2.6+, '
```

Comment:
You are right, I just took the chance to point out those problems since I was reviewing that portion of the code. 

@akien-mga, do you want them fixed in a separate issue or since they are so small can they be fixed on this one?